### ISSUE TITLE
Comment list is always empty

### ISSUE THREAD

**Issue by Feuerfuchs** [posativ/isso#450]
I just switched from the last stable version to the master branch of this repository. Now the comment list API endpoint (`/?uri=...`) is broken for me since it will always return a 404 error. The count API endpoint (`/count?uri=...`) works fine, though.

You can see this problem live here: https://newtest.feuerfuchs.eu/en/blog/2014/05/02/another-hugo-post/
Fetch API result: https://comments.feuerfuchs.eu/?uri=/en/blog/2014/05/02/another-hugo-post/
Count API result: https://comments.feuerfuchs.eu/count?uri=/en/blog/2014/05/02/another-hugo-post/
[Admin interface screenshot](https://user-images.githubusercontent.com/14197353/40931270-bede5a7c-682a-11e8-80b8-b3baab66064a.png)

**Comment by pozitron57** [comment]
Do you have the same problem when there are more than zero comments? See #301, isso returns 404 when there are no comments.

**Comment by Feuerfuchs** [comment]
Yes, that's the point of my issue. On my website, you will see in the article header that there are currently 2 comments for this article (which is the number reported by `/count?uri=...`). However, if you scroll down to the actual comments list, it says "No Comments Yet" (as reported by `/?uri=...`).

I'm absolutely sure the comments must show up here because I posted them myself on this very page and the admin panel shows a matching URI for the comments (see the screenshot I linked).

**Comment by pozitron57** [comment]
Now I see, sorry, didn't read your post attentively enough.

**Comment by Feuerfuchs** [comment]
No problem, my description wasn't quite clear about what the issue actually is. I fixed it now. :)

**Comment by jelmer** [comment]
can you still reproduce this with 0.11.1?

**Comment by Feuerfuchs** [comment]
I will check as soon as possible! It might take a few days, though, since I'm very busy right now.

**Comment by Feuerfuchs** [comment]
That went faster than expected so here we go: The issue is still there, unfortunately.
The comment list reports 404: https://comments.feuerfuchs.eu/?uri=/en/blog/2014/05/02/another-hugo-post/
The counter reports 2: https://comments.feuerfuchs.eu/count?uri=/en/blog/2014/05/02/another-hugo-post/
The admin interface lists 2 comments:
[screenshot_2018-11-26 isso admin](https://user-images.githubusercontent.com/14197353/49016102-5becf380-f185-11e8-9635-5a2e8e7826bc.png)
Isso version is 0.11.1:
![bildschirmfoto vom 2018-11-26 14-13-48](https://user-images.githubusercontent.com/14197353/49016152-7aeb8580-f185-11e8-9e19-b93a383e0e5a.png)

**Comment by ix5** [comment]
Is this still current?
